pytest-embedded-arduino/pytest_embedded_arduino/app.py

```diff
@@ -17,7 +17,14 @@ class ArduinoApp(App):
     """
 
     #: dict of flash settings
-    flash_settings = {'flash_mode': 'dio', 'flash_size': 'detect', 'flash_freq': '80m'}
+    flash_settings = {
+    	'esp32': {'flash_mode': 'dio', 'flash_size': 'detect', 'flash_freq': '80m'},
```

Comment:
here this line is mixed with spaces and tabs

Reply:
Should be fixed by running pre-commit. As a commit message I have pasted what you suggested co you can squash both commits and keep the last RIGHT message. Sorry for that.